tests/EdgeToEdge/Routes/DisplayPageRouteTest.php

```diff
@@ -115,4 +95,35 @@ public function testWhenPageNameContainsSlash_404isReturned(): void {
 
 		$this->assert404( $client->getResponse() );
 	}
+
+	private function getMockPageSelector( string $pageId ): PageSelector {
+		$pageSelector = $this->createMock( PageSelector::class );
+		$pageSelector
+			->method( 'getPageId' )
+			->willReturnArgument( 0 )
+			->with( $pageId )
+			->willReturn( $pageId );
+		return $pageSelector;
+	}
+
+	private function getNotFoundPageSelector( string $pageId ): PageSelector {
```

Comment:
The naming convention used in this codebase is to call functions returning a new instance of a service newSuchService rather than getSuchService

Reply:
Thanks for pointing it out! I was not fully aware we had this this convention (or just forgot we had it). As the one responsible for merging, I've created a followup: https://github.com/wmde/FundraisingFrontend/pull/1230